### ISSUE TITLE
NuScnens dataset splitting

### ISSUE THREAD

**Issue by jiangchaokang** [mit-han-lab/bevfusion#325]
Hello ~ Thank you for your work which has been very helpful to me! I would like to ask a question about data set splitting. I want to train the proposed network on both the training set and the validation set. But I noticed that the dataset created does not have the nuscenes_infos_trainval.pkl file. How can I do to perform the training on both training and validation sets?
I am looking forward to your reply. Good luck!

**Comment by kentang-mit** [comment]
You may directly concatenate the lists in `nuscenes_infos_train.pkl` and `nuscenes_infos_val.pkl` to get the desired file. This is how we create trainval infos.

**Comment by jiangchaokang** [comment]
Loading these two files (```nuscenes_infos_train.pkl``` & ```nuscenes_infos_val.pkl```) generates two dictionaries. I wonder how to merge the two dictionaries properly. Can you release this part of the code for me? Thank you very much!
